doc/classes/PropertyTweener.xml

```diff
@@ -59,7 +78,26 @@
 
 				func tween_curve(v):
 				    return curve.sample_baked(v)
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				[Export]
+				public Curve Curve { get; set;}
+
+				public override void _Ready()
+				{
+				    Tween tween = CreateTween();
+				    // Interpolate the value using a custom curve.
+				    Callable tweenCurveCallable = Callable.From((float value) => TweenCurve(value));
+					tween.TweenProperty(this, "position:x", 300.0f, 1.0f).AsRelative().SetCustomInterpolator(tweenCurveCallable);
```

Comment:
```suggestion
				    tween.TweenProperty(this, "position:x", 300.0f, 1.0f).AsRelative().SetCustomInterpolator(tweenCurveCallable);
```
The CI is complaining about (at least) this line. It needs to be four spaces, not a tab, for indentation within code examples. Note that the indentation for the XML itself is tabs.

Reply:
Oh shoot, are there any other instances of this?

Reply:
I don't see any, but you should double check 🙂 

If you plan on making a habit of contributing, there is a way to run the pre-commit locally if you set it up, which should catch most cases like this. See https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#using-clang-format-locally

Reply:
Worth noting too that some text editors like VS Code let you distinguish between tabs and spaces. In VS Code it's done when you have text selected. Unfortunately the GitHub UI doesn't do that by default.
![Code_WiKm1UeOv3](https://github.com/user-attachments/assets/0160a76c-3761-442c-9303-7403ae9d982e)

Reply:
I edit in VScode and I'm backspacing and doing four spaces on each line of code where I believe this would be an issue.  The line mentioned here is the only one where it is saying that it has been changed from how it was before.

I believe this is the only one that was tabbed rather than four spaced, and I'm considering just clicking the commit suggestion button because this is so small.